_layouts/post.html

```diff
@@ -6,36 +6,41 @@
 {% include feature-image.html %}
 
 <div class="grid-container usa-section">
+  <div class="grid-row grid-gap">
+    <div class="grid-col-2">
+      <a class="back-blog link-arrow-left" href="{{ site.baseurl }}/blog/">
+        {% include svg/icons/arrow-left.svg %}
+        <span class="usa-sr-only">Back to</span>
+        Blog
+      </a>
+    </div>
+    <div class="grid-col-8">
```

Comment:
Could you add the `tablet` prefix here so it stacks on mobile

Reply:
👍